common/src/scheduler/task.rs

```diff
@@ -78,10 +78,6 @@ pub enum ScheduledTask<AccountId> {
     /// Remove gas reservation.
     #[codec(index = 8)]
     RemoveGasReservation(ProgramId, ReservationId),
-
-    /// Remove resume program session.
-    #[codec(index = 9)]
-    RemoveResumeSession(SessionId),
```

Comment:
revert removal of the this task pls, with marking it deprecated and noop on processing for case of being stored in on-chain scheduler storage OR impl check for absence of this variant in storage for migration check

Reply:
basically, its better for now to just revert declaration removal, as we previously did for pause program task

Reply:
Why it was resolved?

Reply:
I've added it back there?
```rust
    #[deprecated = "Paused program storage was removed in pallet-gear-program"]
    RemoveResumeSession(u32),
```
https://github.com/gear-tech/gear/pull/3947/files#diff-adb19c99c74f2899a45fa21410cb0f27ad774066cb663f3d0f2f0e92698cc73dR84

Reply:
A. I thought the question is also about SessionId -> u32